utils/jwtUtil.js

```diff
@@ -56,25 +97,49 @@ function verifyToken(req, cb) {
   }
 
   if (token) {
-    jwt.verify(token, env.tokenSecret, {}, (err, decodedData) => {
+    jwt.verify(token, secret, {}, (err, decodedData) => {
       if (err) {
-        handleInvalidToken(cb);
-      } else {
-        User.findOne({ where: { name: decodedData.username } })
+        return handleInvalidToken(cb);
+      } else { // eslint-disable-line no-else-return
+        return User.findOne({ where: { name: decodedData.username } })
         .then((user) => {
-          // set user in request
           if (user) {
-            req.user = user;
-            return cb();
-          }
+            req.user = user; // set user in request
 
-          return handleInvalidToken(cb);
+            /*
+             * No need to check the token record if this is the default UI
+             * token.
+             */
+            if (decodedData.username === decodedData.tokenname) {
```

Comment:
So, are we restricting users to create tokenname = username? Or we are going to follow the strategy of having an extra field like, `isUIToken: true` when creating token from UI.